lib/console/console.coffee

```diff
@@ -44,11 +44,26 @@ class Console
 
   done: -> @isInput = false
 
-  out: (s) -> @view.add @view.outView(s), @isInput
-
-  err: (s) -> @view.add @view.errView(s), @isInput
-
-  info: (s) -> @view.add @view.infoView(s), @isInput
+  @debounce: (t, f) ->
+    timeout = null
+    (args...) ->
+      if timeout? then clearTimeout timeout
+      timeout = setTimeout (=> f.call this, args...), t
+
+  @buffer: (f) ->
+    buffer = []
+    flush = @debounce 10, ->
```

Comment:
Yup, that's exactly right. It's a pretty arbitrary choice but it seems to capture single `println`s roughly right.